### PR TITLE
[ADD] mail_activity_unlink_log

### PR DESCRIPTION
**Pull request by etobella**
This addon enforces the creation of a message when an activity is unlinked.
This change tries to avoid a possible bad practice of deletion of an activity without notifying the deletion.

### CODE REVIEW

**Review by LoisRForgeFlow — COMMENTED** [review]
LGTM but maybe a more descriptive module name would be good. On a quick shoot, I can think on mail_activity_log_unlink or mail_activity_notify_unlink

**Comment by pedrobaeza** [comment]
Yeah, `mail_activity_unlink_log` seems OK.

File: mail_activity_unlink_log/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2023 CreuBlanca
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Mail Activity Unlink",
```
Comment (LoisRForgeFlow) [inline mail_activity_unlink_log/__manifest__.py]: Change the name here

**Review by LoisRForgeFlow — APPROVED** [review]
:+1:

**Review by sanchonuria — APPROVED** [review]

**Review by kluna1998 — APPROVED** [review]
LGTM! 👍🏽

**Pull request merged**